### PR TITLE
adding in diversity to MF roof insulation. 

### PR DESCRIPTION
**Pull request by JLReyna**
As a quick fix, replicating SF roof insulation distribution

Addresses #[issue number here].

## Pull Request Description

Improve diversity for MF roof insulation by manually copying SF roof insulation (from Insulation Unfinished Attic). In the future, script NAHB / RBSA to pull MF distribution and also improve attic / non-attic split. 

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] PAT project measures/outputs have been updated
- [ ] Documentation has been modified appropriately
- [x] Any new options are added to `project_testing`
- [x] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -997,7 +997,12 @@ Insulation Unfinished Attic	"Roof R-49, Unvented"	ResidentialConstructionsUnfini
...
+Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-19	ResidentialConstructionsFinishedRoof	cavity_r=19	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
 Insulation Finished Roof	R-30	ResidentialConstructionsFinishedRoof	cavity_r=30	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-38	ResidentialConstructionsFinishedRoof	cavity_r=38	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-49	ResidentialConstructionsFinishedRoof	cavity_r=49	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0
```
Comment (ejhw) [inline resources/options_lookup.tsv]: Can you update the `cavity_depth` values to preserve the R-value per inch? (based on the R-30 example: 30/9.25 = 3.2432432432 R/in)

Comment (JLReyna) [reply resources/options_lookup.tsv]: Updated to match BEopt cavity depth = 11.25 inches for R-38 and R-49

File: resources/options_lookup.tsv [hunk 2]
```diff
@@ -998,11 +998,11 @@ Insulation Unfinished Attic	"Roof R-49, Unvented"	ResidentialConstructionsUnfini
 Insulation Finished Roof	None																																														
 Insulation Finished Roof	Uninsulated	ResidentialConstructionsFinishedRoof	cavity_r=0	install_grade=1	cavity_depth=9.25	filled_cavity=false	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
 Insulation Finished Roof	R-7	ResidentialConstructionsFinishedRoof	cavity_r=7	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
-Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=9.25	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0																																					
+Insulation Finished Roof	R-13	ResidentialConstructionsFinishedRoof	cavity_r=13	install_grade=1	cavity_depth=4.01	filled_cavity=true	framing_factor=0.07	drywall_thick_in=0.5	osb_thick_in=0.75	rigid_r=0.0
```
Comment (ejhw) [inline resources/options_lookup.tsv]: @JLReyna Typo? I think this should be cavity_depth=3.5

Comment (JLReyna) [reply resources/options_lookup.tsv]: Good catch. That was the calculated version. BEopt uses 9.25

Comment (ejhw) [reply resources/options_lookup.tsv]: I see 3.5 in BEopt for R-13

**Comment by JLReyna** [comment]
> One comment I have is that, if you're not going to include a "tsv maker" for these tsv files in a companion PR, create a reminder to do this sometime in the future: https://github.com/NREL/resstock-estimation/issues.

Yep. Already done: https://github.com/NREL/OpenStudio-BuildStock/issues/396

**Pull request merged**